helm/templates/hpa.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "substrate-telemetry.fullname" . }}
+  labels:
+    {{- include "substrate-telemetry.labels" . | nindent 4 }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ include "substrate-telemetry.fullname" . }}
```

Comment:
i'm not sure if i understand this correctly. should this be the name of the shard deployment i.e. `telemetry-shard`?

Reply:
the hpa is created by default when you run `helm create ...`, I didn't remove it so we can use it later with some adjustments. If it's necessary, the file could be removed as well.